go.sum

```diff
@@ -1732,8 +1732,9 @@ github.com/vektah/gqlparser v1.1.2/go.mod h1:1ycwN7Ij5njmMkPPAOaRFY4rET2Enx7IkVv
 github.com/vishvananda/netlink v0.0.0-20181108222139-023a6dafdcdf/go.mod h1:+SR5DhBJrl6ZM7CoCKvpw5BKroDKQ+PJqOg65H/2ktk=
 github.com/vishvananda/netlink v1.1.0/go.mod h1:cTgwzPIzzgDAYoQrMm0EdrjRUBkTqKYppBueQtXaqoE=
 github.com/vishvananda/netlink v1.1.1-0.20201029203352-d40f9887b852/go.mod h1:twkDnbuQxJYemMlGd4JFIcuhgX83tXhKS2B/PRMpOho=
-github.com/vishvananda/netlink v1.1.1-0.20220316193741-b112db377d18 h1:nBq2uYJYTG5nfCjOSQk8lkUV98R2YEu4UHw8y4grg3E=
 github.com/vishvananda/netlink v1.1.1-0.20220316193741-b112db377d18/go.mod h1:twkDnbuQxJYemMlGd4JFIcuhgX83tXhKS2B/PRMpOho=
```

Comment:
This version only seems partially removed, you might need to run another tidy.

Reply:
OK. It is strange that it is leaving 1 of the 2 lines. 🤷

Reply:
Maybe because of https://github.com/DataDog/ebpf-manager/blob/main/go.mod#L11